src/main/app/invite-service/InviteService.ts

```diff
@@ -0,0 +1,48 @@
+import { AuthorizedAxios } from '../authorized-axios/AuthorizedAxios';
+import config from 'config';
+import { Logger } from '../../interfaces/Logger';
+import { TelemetryClient } from 'applicationinsights';
+
+enum invitationTypes {
+  INVITE = 'INVITE',
+  SELF_REGISTER = 'SELF_REGISTER',
+  REACTIVATE = 'REACTIVATE',
+}
+
+export class InviteService {
+  private readonly INVITE_ENDPOINT: string = config.get('services.idam.endpoint.invite');
+  private readonly CLIENT_ID: string = config.get('services.idam.clientID')
+
+  constructor(
+    private readonly idamApiAxios: AuthorizedAxios,
+    private readonly logger: Logger,
+    private readonly telemetryClient: TelemetryClient
+  ) {}
+
+  public inviteUser = (
+    email: string,
+    forename: string,
+    surname: string,
+    roles: string[] = [],
+    invitedBy?: string,
+    successRedirect?: string
+  ) => {
+    return this.idamApiAxios
+      .post(this.INVITE_ENDPOINT, {
+        invitationType: invitationTypes.INVITE,
+        email,
+        forename,
+        surname,
+        clientId: this.CLIENT_ID,
```

Comment:
I think this should be the client id of the service inviting the user (so, adoption for private beta)

Reply:
Addressed in https://github.com/hmcts/idam-user-dashboard/pull/440/commits/e2eb3d1c2f1531add059105a5dc9196c92ecaff7